Cargo.lock

```diff
@@ -2987,15 +2987,16 @@ checksum = "e6d5a32815ae3f33302d95fdcb2ce17862f8c65363dcfd29360480ba1001fc9c"
 [[package]]
 name = "gateway-messages"
 version = "0.1.0"
-source = "git+https://github.com/oxidecomputer/management-gateway-service#e71c87a6ee63e8f8d6a1940cfeda00e0c312fa19"
+source = "git+https://github.com/oxidecomputer/management-gateway-service#ea3225658f718b12cd58d611960e0a936be47d0d"
```

Comment:
This needs to be the final commit we merge and I don't think the MGS side is ready yet